atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -48,26 +49,34 @@ def __init__(self, tasker, workflow,
         :param retry_delay: int, seconds between pull attempts
         :param insecure: bool, allow non-https pull if true
         :param secret: str, path to secret
+        :param expect_v2schema2: bool, retry until Pulp retries schema 2 digest
```

Comment:
I don't understand this description

Reply:
How about `require Pulp to return a schema 2 digest and retry until it does`?

Reply:
Fixed